tests/tests_sound_power/test_sound_power_level_iso_3744.py

```diff
@@ -403,6 +407,21 @@ def test_sound_power_level_iso_3744_get_thirdoctave_center_frequencies(dpf_sound
     assert fc_3[12] == pytest.approx(EXP_FC_3_12)
 
 
+def test_sound_power_level_iso_3744_load_project_with_calibrations(dpf_sound_test_server):
```

Comment:
@anshlachamb should we keep the test "test_sound_power_level_iso_3744_load_project(...)" as this new one is doing the same thing on a more appropriate project (with signals that have calibration != 1)?

Reply:
@anssmolla the original test method is also testing the names of the signals, and some other values

Reply:
can't we do it all in a same test?